proto/connectrpc/conformance/v1alpha1/service.proto

```diff
@@ -60,66 +73,116 @@ service ConformanceService {
   // If the input stream is empty, the server should send a single response
   // message that includes no data and only the request properties (headers,
   // timeout).
-  rpc BidiStream(stream BidiStreamRequest) returns (stream ConformancePayload);
+  //
+  // Note that clients could specify a list of desired responses that is less than
+  // or greater than the amount of requests they send on the stream. If this is the case
+  // and full duplex is specified then servers should behave as follows:
+  //
+  // If the total desired responses is less than the requests, the server should
+  // simply stop sending responses once the total desired is sent and continue to read
+  // messages until the client is finished sending.
+  //
+  // If the total desired responses is greater than the requests, the server should
+  // flush the remaining responses once the client is finished sending.
+  //
+  // Note that unequal desired responses and requests is irrelevant with half duplex calls.
+  rpc BidiStream(stream BidiStreamRequest) returns (stream BidiStreamResponse);
 }
 
-message UnaryRequest {
-  // response headers to send
+// A definition of a response to be sent from a single-response endpoint.
+// Can be used to define a response for unary or client-streaming calls.
+message SingleResponseDefinition {
```

Comment:
`Single...` meh. Open to other suggestions. Using `Unary` felt incorrect/misleading here since this could be used for client streams also.

Reply:
I think `Unary` would be better. In my mind, a client stream does have a unary response (just not a unary request) 🤷